core/templates/dev/head/pages/exploration_editor/editor_tab/state_editor_responses.html

```diff
@@ -99,10 +99,8 @@
       </button>
     </div>
   </md-card>
-  <md-card style="margin: 20px 0px; padding: 0px;" ng-if="!isCurrentInteractionTrainable()">
-{% else %}
-  <md-card style="margin: 0px; padding: 0px;">
 {% endif %}
+  <md-card style="margin: 0px; padding: 0px;">
```

Comment:
Nit: use 0 instead of 0px (it was wrong before).

This still works well in both cases, right (where the feconf flag is turned on or turned off)?